pyzipcode/__init__.py

```diff
@@ -157,15 +166,30 @@ def find_zip(self, city=None, state=None):
 
         return format_result(self.conn_manager.query(ZIP_FIND_QUERY, (city, state)))
 
-    def get(self, zipcode):
+
+    def get(self, zipcode, default=None):
         """
         Return a list of one ZipCode object for the given zipcode.
         """
-        return format_result(self.conn_manager.query(ZIP_QUERY, (zipcode,)))
+        result = format_result(self.conn_manager.query(ZIP_QUERY, (zipcode,)))
+        if result:
+            return result
+        return default
 
     def __getitem__(self, zipcode):
         data = self.get(str(zipcode))
         if data is None:
-            raise IndexError("Couldn't find zipcode: '%s'" % zipcode)
+            raise KeyError(f"Couldn't find zipcode: '{zipcode}'")
```

Comment:
This is a backwards-incompatible change, but to me it makes more sense for this to act as a dictionary, since this data isn't a list. You can't do `zcdb[0]`